gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -76,8 +76,18 @@ def __init__(self,
             '--temp_location {}'.format(parser_args.temp_location),
             '--job_name {}'.format(
                 ''.join([test_name, suffix]).replace('_', '-'))]
+
+    self._header_blob_name = ''
+    self._header_path = ''
     for k, v in kwargs.iteritems():
-      args.append('--{} {}'.format(k, v))
+      if k == 'header_blob_name':
```

Comment:
I wonder if adding `header_blob_name=None` as part of the init arguments would work here. If not, consider adding an explicit code like this:
```
if 'header_blob_name' in kwargs:
  self._header_blob_name = self._append_suffix(kwargs['header_blob_name'])
  args.append(...)
  del kwargs['header_blob_name']
continue with the rest of the code as before.
```